### DISCUSSION TITLE
Frigate not working anymore

### DISCUSSION THREAD

**Discussion by Lulugaot** [blakeblackshear/frigate#14539] [Ask A Question] [answered]
Was working fine since about early August. It stopped working on our about October 18th or so. Tried to figure out the cause but I could not get it to work. Deleted all recordings, the days worth, which where on my NAS (OMV) to see if that would help. USB Coral not flashing, Gave up and uninstalled frigate. Noticed all entities still showing up in home assistant etc. Reinstalled frigate, and it would still not start. Deleted db folder in home assistant, tried again, would not start. Uninstalled, then installed, created db folder because new installation does apparently not create any of its folders. Deleted deprecated no longer supported entities etc, retried fresh install, still not working. USB Coral light still on indicating power but not flashing like it used to upon detection etc. Tried my old home assistant backups and it still would not work. I gave up after spending about eight hours trying to get it working the way it was before. I believe that when uninstalling frigate it should remove all the entities associated with it's function and when you reinstall it it should create all its new dependent folders necessary for its proper operation with a fresh install. In addition it should maybe try to signal / or communicate with / or reset the Google Coral to activate or ready it with the new install. I unplugged my Coral and frigate still would not start. When plugged back in Coral still not flashing. My system is Pi5 with official Pi NVME hat with a 1TB Corsair MP600 Core Mini, Pi5 active cooler, USB Google Coral, running HAOS only. My NAS is Pi3 v1.2 with two (2) USB connected T-Force Vulcan Z 2.5 inch SSD of 1TB each. This is used and setup through the home assistant on the Pi5 for storage of media, data and backups using this NAS, running OMV 6. There has to be a solution in the future for a comprehensive Frigate fresh install with all of its pertinent necessary folders and system into home assistant, overwriting and or deleting existing unnecessary bits to help with a fresh start. I am pretty well versed on computers and have been dealing with them for years, and I'm an IT support professional, built my own computers and setup wired network in home for all to use etc. Over the years ran Red Hat, BSD, Ubuntu, Windows, print servers, RAID setups etc. All for me, our home, and to help the kids through school and to familiarize everyone with computers and have never stopped since those first days with floppy disks and the Commodore 128. Sorry for the long post, I am giving up on Frigate until maybe a new release that reinstalls everything needed and removes all not needed comes along. I will have to  rely on my Reolink NVR which I bought 5 months ago and continues to operate just fine. I wanted Frigate as a backup and loved the features and integration it has with home assistant and my existing smart home. Thank you !

**Comment by NickM-27** [comment]
We can't help without logs and the config

**Comment by Lulugaot** [comment]
[FrigateYaml.pdf](https://github.com/user-attachments/files/17494675/FrigateYaml.pdf)
[logs.pdf](https://github.com/user-attachments/files/17494676/logs.pdf)
here they are

**Reply by hawkeye217** [reply]
The logs show:

```
2024-10-23 03:40:21.689537994 [2024-10-22 23:40:21] frigate.app INFO :
Creating directory: /media/frigate/recordings
2024-10-23 03:40:21.695262506 [Errno 13] Permission denied: '/media/frigate/recordings
```

You need to check the permissions on the directory.

**Reply by Lulugaot** [reply]
How exactly do you do that ? And where, which directory name etc. How can I acquire / get access in order to accomplish this to see if this change will work ? Remember everything was working fine until about a few days or a week ago , no changes were done and it stopped working. Could it be a bug, a breaking change, a hacking code sent out to home assistant users etc. (Supposed to be stand alone OS) I am totally lost. Appreciate the help but without a step by step guide of what I should try I am lost. Started this adventure in late May early June with home assistant on my Pi 3 and it worked after learning and playing with YAML and HAOS. Upgraded to Pi 5 and all necessary accompanying hardware, went through hoops to get it all configured and working since mid August and repurposing the Pi 3 as my NAS for use with home assistant only. Thank you for reading and appreciate your time and effort but I don't know where and how to start or if I will be able to actually access the actual stuff I have to change since HAOS doesn't let you access things on a certain level. Or OMV for that matter. Thanks again will continue to monitor for any updates. Was thinking of trying the Homie app (the one on YouTube videos) that say "homie is not your friend" which apparently removes stuff from home assistant that is no longer needed or deprecated etc.

**Reply by hawkeye217** [reply]
There are too many variables at play to be able to give you a step-by-step on how to fix your exact problem with your setup, and much of it is beyond the scope of Frigate itself. Also, a lot depends how you've installed Frigate. If you are using HAOS and installed the Frigate HA addon, you should be able to completely remove the addon and reinstall it. if you've configured NAS in HA, you'll need to refer to the Home Assistant documentation on how to configure and use that.

**Comment by Lulugaot** [comment]
"utf-8 codec can't decode byte etc." listed as only data in various (quite a few) home assistant folders / directories when viewing using the 'file editor' add-on.

**Comment by Lulugaot** [comment]
Thank you all for your help and advice. At the moment I can't figure out how to wipe and reformat my HAOS NVME drive to start over from scratch so that I can have a brand new unaltered home assistant on the NVME to start with to which I can install mqtt, then frigate and start fresh and see if it will work like it did for the past two months. There is no option to reinstall the complete OS (home assistant) to start from scratch. The backups (full) that I have, apparently restore certain items even though it's supposed to be a full restore and when I check HA , some old data is still there in folders and other components. I will remove frigate like I did previously ( entities still remain listed on dashboard) and probably not use home assistant as much as before, if not probably anymore. Have to cancel Nabu Casa subscription as well. Thank you again. By the way everything was working fine but my storage was starting to get full so I put together my NAS as stated and had been using that for some time, about a month or a little more and had no issues. No crazy variables, just Pi 5 running only home assistant , frigate add-on and Pi 3 running OMV 6 as my NAS storage for frigate media as instructed to do, to solve the storage problem. Apparently configured correctly since it ran flawless for two months. I thank you all and this home assistant chapter in my home will probably be ending now. It's been a good journey learning and using YAML etc. , bought a few items , sky connect, one Aqara P1, one Aqara FP2, 2TB SSD, but no more. Thank you all again for your time !

**Comment by Lulugaot** [comment]
This issue is up in the air like many others, with no simple solutions. I just have a simple setup without a million other add-ons or integrations other than home assistant, HACS, mqtt, frigate integration and frigate notification automation installed on my Pi 5 with Pi NVME hat and a Google Coral. Unfortunately Frigate system stopped working after successfully running for over a month with no hiccups. Everyone needs to look at the logs but the logs don't really make it obvious what is causing issues and the needed actions to repair these issues is far from clear, even with my better than average computer abilities. A lot of time has been spent to try and get it working again with no luck, utf-8 code error  displayed in various system folders so something is not right. I want to thank Blake Blackshear for all his time and effort to develop Frigate and all others with their assistance in this matter. Time to uninstall.

**Comment by Lulugaot** [comment]
I want everyone to know that I regrouped and had a moment to find time and try something else. Changed detection from Coral to CPU detection first, then removed the frigate network storage from HAOS , tried again and frigate started to work. I then plugged in the Coral USB and changed detection back to the USB instead of CPU detection and it all seems to be working just fine now. The only thing I can't do after trying many times is to reestablish network storage for frigate so that my NVME is not filled up with recordings , running out of space eventually. Will work on reestablishing the NAS storage for frigate once I can figure out how to overcome the " can't connect, something about bind and or media folder contains files etc. There should be a quick way to select by a drop-down or checkbox to easily move the frigate storage to a network location such as your NAS device without having to try and figure out how and where and what steps to take, trying to find what files have to be saved or folders to rename in order to create a frigate folder and try and reload existing recordings back to this folder or folders. It is a pretty deep (not easy) process to try and figure out not to mention the errors that pop up in attempting to do this. I will keep trying. Might have to wipe my OMV NAS on my other Pi to try and start from scratch and see if that helps accomplish this. I continue to read a lot of documentation and revisit many youtube videos to try and do this all over again into a working configuration. But I am glad that at least Frigate is working now, just have to move the storage from the only storage on my Pi 5 NVME to my Pi 3 OMV NAS as it was before it self destructed for no apparent reason. Thanks everyone !

**Comment by Lulugaot** [comment]
Check out the latest info. Deletion of OMV network storage in HAOS worked for my setup !

**Reply by Lulugaot** [reply, marked as answer]
Has been running fine, also wipe of and fresh install of NAS folders and HAOS network storage connections deletion and fresh add of storage connections, now everything working just fine on my Pi 5 HAOS with Coral USB, Pi 3 NAS, two Reolink POE cameras ( RLC-843A  and CX-410). The 843A is H265 codec and the 410 is H264 codec. Bought another 410 and will be adding it soon. Still deciding on the fourth and last camera to buy which will also be a Reolink, probably one of their PTZ versions , the Trackmix. If it ain't broke, don't fix it. Too many requests for adding a whole bunch of stuff and people having a lot of issues with their setups. I believe Frigate is fine and serves it's purpose. As far as issues, I think people have to try, review, change and debug their configurations and errors (Proxmox, Docker, etc.) to see if they can run it on their own specific hardware. I know it takes countless hours sometimes but it is worth it in the end. It is Linux and sometimes it doesn't play nice with x-86 and the plethora of motherboards and NUCS just like the differences between Windows, Mac and Linux software and systems. But I'm a happy camper with my Pi SBC's in my home lab (all "arm" based) working without a hitch. Thank you ! If at first you don't succeed try try again.
